rq_function.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import os
+
+def add(a,b):
+    print(a+b)
+
+def count_lines(path):
+    file = open(path, "r")
+    line_count = 0
+    for line in file:
+        if line != "\n":
+            line_count += 1
+    file.close()
+    print(line_count)
+
+def poppunk_assign(path):
+    os.system('''
+    cd %s
+    poppunk_assign --db GPS_v4 --query input_data/qfile.txt --output poppunk_clusters_flask --threads 4
+    '''
+    %path)
```

Comment:
Is poppunk a python library? Can you call it without going through the command line script?

Reply:
It is written in Python, but I just found the documentation on how to run it in the command line. Not sure how I could access it directly in python?